examples/standalone/README.md

```diff
@@ -1,11 +1,11 @@
 # Standalone example
 
 This example:
-- Presents the deploy of the Secured Data Warehouse blueprint variant for ingesting encrypted data from on-prem sources.
+- Presents the deploy of the blueprint for ingesting encrypted data from on-premisses environments.
```

Comment:
Typo - Presents the deploy of the blueprint for ingesting encrypted data from on-premises environments

Reply:
fixed